src/context.js

```diff
@@ -190,6 +192,37 @@ function EvolvContext() {
     ensureInitialized();
     return key in remoteContext || key in localContext;
   };
+
+  /**
+   * Adds value to specified array in context. If array doesnt exist its created and added to.
+   *
+   * @param key The array to add to.
+   * @param value Value to add to the array.
+   * @param local {Boolean} If true, the value will only be added to the localContext.
+   * @param limit {Number} Max length of array to maintain.
+   * @returns {boolean} True if value was successfully added.
+   */
+  this.pushToArray = function(key, value, local, limit) {
+    limit = limit || DEFAULT_QUEUE_LIMIT;
+
+    ensureInitialized();
+
+    const context = local ? localContext : remoteContext;
+    const originalArray = objects.getValueForKey(key, context);
+
+    let newArray;
+    if (originalArray) {
+      newArray = originalArray.slice();
+      newArray.push(value);
+      while (newArray.length > limit) {
+        newArray.shift();
+      }
```

Comment:
Can you explain this? I read it as you are dropping items in the beginning of the array if it exceeds the default queue size? But why? Is it okay to discard those items?

Reply:
Ok, it's acting as a queue so we don't end up with a huge list in the case of some looping error. Users can set an arbitrarily large limit if they want it to be big.

Reply:
Try this first, but I think you can also achieve the same behavior with this:
```
const combined = (originalArray || []).concat([value]);
const newArray = combined.slice(combined.length - limit);
```

Reply:
Done.